pom.xml

```diff
@@ -206,7 +206,7 @@
         <artifactId>derby</artifactId>
         <!-- we should use whatever hive-metastore is using. Cannot rely
           on the transitive dep because we want it test scoped. -->
-        <version>10.10.2.0</version>
+        <version>10.14.2.0</version>
```

Comment:
As the comment points out, this should match the `hive-metastore` version so I think we should decline and close this PR.